pkg/gofr/datasource/opentsdb/interface.go

```diff
@@ -18,11 +25,6 @@ type connection interface {
 	SetWriteDeadline(t time.Time) error
 }
 
-// HTTPClient is an interface that wraps the http.Client's Do method.
-type httpClient interface {
-	Do(req *http.Request) (*http.Response, error)
-}
```

Comment:
For record, you moved http client interface before the connection one.

If you move it back, the diff will be smaller